irods/tests/tasks/test_install_plugins.yml

```diff
@@ -11,46 +11,49 @@
     - irods-microservice-plugin-netcdf
     - irods-api-plugin-netcdf
 
-- name: test ensure netcdf rpms removed
-  stat:
-    path: /root/{{ item }}-1.0-centos{{ ansible_distribution_major_version }}.rpm
-  register: response
-  failed_when: response.stat.exists
-  with_items:
-    - irods-icommands-netcdf
-    - irods-microservice-plugin-netcdf
-    - irods-api-plugin-netcdf
+# # XXX - netcdf is no longer accessible for iRODS 4.2.8, needs to be readded when upgrading to iRODS 4.2.11
+# - name: test ensure netcdf rpms removed
+#   stat:
+#     path: /root/{{ item }}-1.0-centos{{ ansible_distribution_major_version }}.rpm
+#   register: response
+#   failed_when: response.stat.exists
+#   with_items:
+#     - irods-icommands-netcdf
+#     - irods-microservice-plugin-netcdf
+#     - irods-api-plugin-netcdf
```

Comment:
The tests `test ensure netcdf plugins not installed` and `test ensure netcdf rpms removed` can be removed since the tasks being tested are being removed.